src/cmd/flux-hostlist.py

```diff
@@ -31,7 +31,8 @@
 stdin, '-'   read a list of hosts on stdin
```

Comment:
commit message for 18e965050a38bc9a56e4477fd3660c6c30735e47: s/excplicitly/explicitly/

Reply:
Thanks! Fixed that and will set MWP.